plugins/reports/frontend/public/javascripts/countly.views.js

```diff
@@ -767,6 +767,22 @@ window.ReportingView = countlyView.extend({
     }
 });
 
+app.addReportsCallbacks = function(plugin, options){
+    if(!this.reportCallbacks){
+        this.reportCallbacks = {};
+    }
+
+    if(!this.reportCallbacks[plugin]){
+        this.reportCallbacks[plugin] = {};
+    }
+
+    Object.assign(this.reportCallbacks[plugin], options);
```

Comment:
Unfortunately Object.assign has no IE support at all, we want to keep compatible with at least latest IE versions too